### PR TITLE
RTL Example: Cover

### PR DESCRIPTION
**Pull request by aqeelat**
Was bored in my first day at a new job so I RTLed the cover template.

I don’t know where this should be placed in the examples page so I didn’t mess around with that.

### CODE REVIEW

File: site/content/docs/5.0/examples/cover/cover.rtl.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai-light/dubai-light.woff") format("woff");
+}
+@font-face {
+  font-family: Dubai;
+  font-style: normal;
+  font-weight: 500;
+  font-display: swap;
+  src: url("https://www.fontstatic.com/fonts/dubai/dubai.eot?#iefix") format("eot"), url("https://www.fontstatic.com/fonts/dubai/dubai.woff") format("woff");
+}
```
Comment (ffoodd) [inline site/content/docs/5.0/examples/cover/cover.rtl.css:19]: Same comment as for the pricing example: is this really needed?

File: site/content/docs/5.0/examples/cover/cover.rtl.css [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+/*
+ * Header
+ */
+
+.nav-masthead .nav-link {
+  padding: .25rem 0;
+  font-weight: 700;
+  color: rgba(255, 255, 255, .5);
+  letter-spacing: .07em;
```
Comment (ffoodd) [inline site/content/docs/5.0/examples/cover/cover.rtl.css:53]: I thought `letter-spacing` was usually a bad idea for Arabic content: is this required by the custom font you used?

**Review by ffoodd — COMMENTED** [review]
## To-do

Simple notes to avoid forgetting anything:

- [ ] add entry in `site/data/examples.yml`
- [ ] add screenshots
- [ ] if a custom `.rtl.css` file is needed, it should be added in `css-prefix-examples-rtl` script (in `package.json`) which handles a glob pattern.

File: site/content/docs/5.0/examples/cover/cover.rtl.css (hunk 1 above)
Comment (XhmikosR) [reply site/content/docs/5.0/examples/cover/cover.rtl.css:19]: I'd rather not add such assets, so that's a hard no from me.

Comment (AqeelAT) [reply site/content/docs/5.0/examples/cover/cover.rtl.css:19]: I was going to use a single import statement for the fonts, however, even though they're the same family, their `font-family` where different (i.e. Dubai, Dubai-light, Dubai-heavy ...), which would make it unintuitive when using `<strong>` and `<small>` or the typed font sizes. I contacted the guy who maintains the CDN (https://www.fontface.me/) to fix this issue and he agreed to do it but I'm still waiting for him to publish the changes.

Comment (XhmikosR) [reply site/content/docs/5.0/examples/cover/cover.rtl.css:19]: The problem is we'd rather not use such CDNs. We only use Google Fonts for fonts. So, if we can skip adding this whole thing, that would be the best.

File: site/content/docs/5.0/examples/cover/cover.rtl.css (hunk 2 above)
Comment (AqeelAT) [reply site/content/docs/5.0/examples/cover/cover.rtl.css:53]: https://twitter.com/Aqeel_AT/status/1363501583774978049

The issue is some fonts overlaying the individual glyphs. Therefore, when you make a text transparent, the overlayed areas become darker. This should not be an issue with the other examples. This one is a special case.

File: site/content/docs/5.0/examples/cover/cover.rtl.css (hunk 1 above)
Comment (AqeelAT) [reply site/content/docs/5.0/examples/cover/cover.rtl.css:19]: I'll try and replace them with ones from Google Fonts.

**Comment by mdo** [comment]
Appreciate the effort here, but unsure how useful it is to RTLify every example, or even these couple? I don't think we need more than a few—it quickly becomes extra work to maintain them all when one is changed.

**Comment by ffoodd** [comment]
@AqeelAT Thanks for your contribution but I'm closing this: we aren't able to fix and maintain new exemples at the moment.

**Pull request closed without merging**